db/migrate/20111103131805_create_sitemaps.rb

```diff
@@ -4,7 +4,7 @@ def self.up
       t.string :url
       t.references :affiliate
       t.timestamp :last_crawled_at
-      
```

Comment:
As this file isn't related to your changes, I would leave it as the version on master (even if that means leaving some extra whitespace):

`git co master db/migrate/20111103131805_create_sitemaps.rb` 

As a general rule, migration files are never altered once they are used; they essentially serve as historical documentation of DB changes over time. (Some organizations do periodically delete old migration files, but we do not.)